heroku/cli
# Remove co

Removes [co](https://www.npmjs.com/package/co), the library we previously needed to achieve an `async/await` like syntax using generators and `yield`. Our current supported node.js version range is `^10.0 || ^12.0 || ^14.0`, which allows us to use native `async-await` built into Node instead of using `co`. Removing `co` will make debugging easier and stack traces nicer when testing or when we get error reports from customers.

I ran the [co-to-async](https://github.com/albinekb/co-to-async) codemod over the codebase to help achieve this.

## How to verify?

* CI should pass. Our test suite seems to have caught a lot of the cases the codemod couldn't convert automatically
* Pull down the branch locally and run a few commands with `bin/run` in the `packages/cli` directly. It should work
